vendor/pacman/js/game.js

```diff
@@ -23,52 +23,81 @@ var TIME_FRUITS = 0;
 
 var HELP_DELAY = 1500;
 var HELP_TIMER = -1;
-			
-function blinkHelp() { 
-	if ( $('.help-button').attr("class").indexOf("yo") > -1 ) { 
+
+let gameOver = new Event("gameOver");
```

Comment:
Maybe consider renaming to something like `gameOverEvent` to avoid confusion with the `gameover()` function.